editor/components/post-taxonomies/flat-term-selector.js

```diff
@@ -174,6 +174,10 @@ class FlatTermSelector extends Component {
 		const termRemovedLabel = sprintf( _x( '%s removed', 'term' ), singularName );
 		const removeTermLabel = sprintf( _x( 'Remove %s', 'term' ), singularName );
 
+		if ( ! hasAssignAction ) {
```

Comment:
It's a tiny micro-optimisation, but this check could happen immediately after `hasAssignAction` is declared, to avoid the processing needed to declare all the other local variables.

Reply:
ac803ad85dc5dfd2cbe70a92f5fb09cf2483deb7